java-checks/src/test/files/checks/DoublePrefixOperatorCheck/DoublePrefixOperatorCheck.java

```diff
@@ -0,0 +1,31 @@
+class test1 {
+  int a = 1;
+  boolean flag = true;
+ 
+  boolean flag2 = !!flag;  // Noncompliant  {{Remove multiple operator prefixes.}}
+  int a1 = ~~~a; // Noncompliant {{Remove multiple operator prefixes.}}
+
+  boolean flag3 = !!!flag; // Noncompliant {{Remove multiple operator prefixes.}}
+  int a2 = ~~a; // Noncompliant {{Remove multiple operator prefixes.}}
+
+  boolean flag4 = !!!flag2; // Noncompliant [[sc=19;ec=21]] {{Remove multiple operator prefixes.}}
```

Comment:
this one is the same as line 8, with a different identifier. I would remove it, or replace it by something else, such as a method invocation for instance:
`boolean flag4 = !!!foo();`